provisioning/device/src/Transports/Http/HttpContentExtensions.cs

```diff
@@ -15,50 +15,34 @@ internal static class HttpContentExtensions
     {
         internal static async Task CopyToStreamAsync(this HttpContent content, Stream stream, CancellationToken cancellationToken)
         {
-#if NET5_0_OR_GREATER
```

Comment:
iirc this extension class was created to support cancellation tokens for http APIs. Now that we no longer support .NEt 5.0, we can get rid of this extension class and call the .NET APIs directly

Reply:
will follow up with an update and test accordingly.